refresh/templates/common/Application.swift

```diff
@@ -9,45 +9,80 @@ import <%- generatedModule %>
 <% if (healthcheck) { -%>
 import Health
 <% } -%>
+<% if (Object.keys(appInitCode.services).length > 0) { -%>
```

Comment:
Seems strange to have this check here but not at the other places where services-related loops are used further down.

Reply:
see previous comment

Reply:
I think there's some confusion, your previous comment was in reply to me asking about `Route.swift.hbs` but this is `Application.swift` -- wires have been cross I think. That said, it makes more sense in the context of `Application.swift`!